config.yml

```
@@ -28,3 +28,117 @@ paths:
+  level: "INFO"  # Can be DEBUG, INFO, WARNING, ERROR, CRITICAL
+  format: "%(asctime)s %(levelname)s %(message)s"
+  show_blocked_requests: true
+  show_allowed_requests: false  # Set to true to see all allowed requests in debug mode
+
+# Request Blocking Patterns
+block_patterns:
+  # Original patterns
+  - "api.hire.toggl.com/graph/candidate"
+  - "/api/candidatures/anticheating/"
+  - "sentry_key="
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)